decryptor/postgresql/pg_general_decryptor.go

```diff
@@ -279,6 +288,9 @@ func (decryptor *PgDecryptor) DecryptBlock(block []byte) ([]byte, error) {
 }
 
 func (decryptor *PgDecryptor) CheckPoisonRecord(reader io.Reader) (bool, error) {
+	if !decryptor.IsPoisonRecordCheckOn() {
+		return false, nil
+	}
 	// check poison record
 	poisonKeypair, err := decryptor.keyStore.GetPoisonKeyPair()
 	if err != nil {
```

Comment:
please change log msg from line 
https://github.com/cossacklabs/acra/pull/215/files#diff-425fc489e15f15caf5c0e25a7490445dR303

decryptor.logger.Warningln("Recognized poison record")

(capital R)

for consistency of logs among services